gammapy/data/observations.py

```diff
@@ -0,0 +1,569 @@
+        Observation stop time
+    telescope_ids : list of int
+        Telescope IDs of participating telescopes
+    target_radec : `~astropy.coordinates.SkyCoord`
+        Target RA / DEC sky coordinates
+    observatory_earth_location : `~astropy.coordinates.EarthLocation`
+        Observatory location
+    muoneff : float ?
+        Observation muon efficiency
+    meta : `~collections.OrderedDict`
+        Dictionary to store metadata
+
+    """
+    def __init__(self, obs_id=None, gti=None, events=None, aeff=None, edisp=None, psf=None, bkg=None,
+                 pointing_radec=None, pointing_altaz=None, pointing_zen=None, observation_time_duration=None,
+                 observation_live_time_duration=None, observation_dead_time_fraction=None, tstart=None, tstop=None,
+                 telescope_ids=None, target_radec=None, observatory_earth_location=None, muoneff=None, meta=None):
+        self.obs_id = obs_id
+        self.gti = gti
+        self.events = events
+        self.aeff = aeff
+        self.edisp = edisp
+        self.psf = psf
+        self.bkg = bkg
+        self.pointing_radec = pointing_radec
+        self.pointing_altaz = pointing_altaz
+        self.pointing_zen = pointing_zen
+        self.observation_time_duration = observation_time_duration
+        self.observation_live_time_duration = observation_live_time_duration
+        self.observation_dead_time_fraction = observation_dead_time_fraction
+        self.tstart = tstart
+        self.tstop = tstop
+        self.telescope_ids = telescope_ids
+        self.target_radec = target_radec
+        self.observatory_earth_location = observatory_earth_location
+        self.muoneff = muoneff
+        if not meta:
+            meta = OrderedDict()
+        self.meta = meta
+
+    def __str__(self):
+        """Generate summary info string."""
+        ss = 'Info for OBS_ID = {}\n'.format(self.obs_id)
+        ss += '- Number of events: {}\n'.format(len(self.events.table) if self.events else 'None')
+        ss += '- Number of good time intervals: {}\n'.format(len(self.gti.table) if self.gti else 'None')
+        ss += '- Type of eff. area: {}\n'.format(type(self.aeff))
+        ss += '- Type of energy disp.: {}\n'.format(type(self.edisp))
+        ss += '- Type of PSF: {}\n'.format(type(self.psf))
+        ss += '- Start time: {:.2f}\n'.format(self.tstart.mjd if self.tstart else 'None')
+        ss += '- Pointing pos: RA {:.2f} / Dec {:.2f}\n'.format(
+            self.pointing_radec.ra if self.pointing_radec else 'None',
+            self.pointing_radec.dec if self.pointing_radec else 'None')
+        ss += '- Observation duration: {}\n'.format(self.observation_time_duration)
+        ss += '- Dead-time fraction: {:5.3f} %\n'.format(100 * self.observation_dead_time_fraction)
+
+        return ss
+
+    # dcfidalgo: I think this method should be moved outside of this class, in some generic PSFMaker ...
+    def make_psf(self, position, energy=None, rad=None):
```

Comment:
Do you need to add `make_psf` in this PR, i.e. is it called?
I would rather relocate it from the `DataStoreObservation` in a separate PR than to introduce a copy here, if possible.

Reply:
Done, i plan to make `DataStoreObservationCTA` a child class of `ObservationCTA`.
So, most likely i will reintroduce it in the next PR, or maybe we will agree on a new place for this method before.